Automattic/jetpack
# Add/installation to my jetpack

Adds the possibility to install the Boost plugin from My Jetpack

#### Changes proposed in this Pull Request:

* Adds the possibility to install the Boost plugin from My Jetpack

#### Jetpack product discussion


N/A

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Setup a JN site
* Install and connect Jetpack
* Make sure Boost is NOT installed
* Go to My Jetpack
* Click to activate Boost
* Confirm boost is activated
* Confirm the plugin was installed
* Deactivate the Boost plugin
* Go to My Jetpack and activate it again
* Confirm the plugin gets re-activated without errors
